docs/manual_doc/Statement_Syntax/Data_Definition_Statements/ALTER_TAG_EDGE_Syntax.md

```diff
@@ -0,0 +1,28 @@
+```
+ALTER {TAG | EDGE} tag_name|edge_name
+    [alter_definition [, alter_definition] ...]
+    [ttl_definition [, ttl_definition] ... ]
+  
+alter_definition:
+| ADD  (prop_name data_type)
+| DROP   (prop_name)
+| CHANGE (prop_name data_type)
+
+ttl_definition:
+    TTL_DURATION = ttl_duration, TTL_COL = prop_name
+```
+>ALTER statement results in multple versions. It's a UNDEFINED behavior if you interprete data with the mis-match version/schema.   
+
+ALTER statement changes the structure of a tag or edge. For example, you can add or delete properites, change the type of an existing property, or rename properties. You can also set a property as TTL, or change the TTL duration.
+Multiple ADD, DROP, CHANGE clauses are permitted in a single ALTER statements, separated by commas. But do NOT add, drop, change the same property in one statements (If you have to do so, make it one of each cluase per ALTER statement. 
```

Comment:
(If you have to do so, make it one of each cluase per ALTER statement. )
")" is missing